portality/view/jct.py

```diff
@@ -0,0 +1,34 @@
+from flask import Blueprint, make_response
+from flask_login import current_user, login_required
+from flask import abort
+from portality.decorators import api_key_required
+
+from portality import models as models
```

Comment:
Hi Richard, or shall I call you... Richard?

Reply:
ha, weird, that's just copied from somewhere else in the code, who knows how it wound up like that originally